src/proto/grpc/testing/control.proto

```diff
@@ -92,28 +73,82 @@ message SecurityParams {
 message ClientConfig {
   // List of targets to connect to. At least one target needs to be specified.
   repeated string server_targets = 1;
-  ClientType client_type = 2;
+
   SecurityParams security_params = 3;
-  // How many concurrent RPCs to start for each channel.
-  // For synchronous client, use a separate thread for each outstanding RPC.
-  int32 outstanding_rpcs_per_channel = 4;
+  // Maximum limit of how many concurrent RPCs in flight at any time
+  // for each channel.
+  // For synchronous clients, typically use a separate thread of control
+  // for each outstanding RPC.
+  // In practice, this parameter can limit QPS for closed-loop and open-loop
+  // tests. This value should be set to at least the Little's law value for your
+  // desired QPS: namely at least QPS/channel * message latency .
+  int32 max_outstanding_rpcs_per_channel = 4;
+
   // Number of independent client channels to create.
   // i-th channel will connect to server_target[i % server_targets.size()]
   int32 client_channels = 5;
-  // Only for async client. Number of threads to use to start/manage RPCs.
+
+  // Allow the driver to specify the number of threads to use to
+  // start/manage RPCs. This only applies in cases where the client thread
+  // count is not dependent on the RPC width or depth (e.g., the async client
```

Comment:
+1 to adding an explanatory comment, but I am actually not sure if this brings more clarity. The way I read it, it says "it's complicated" :-) Not sure if this can be improved though.